docs/content/querying/lookups.md

```diff
@@ -27,6 +27,13 @@ and such data belongs in the raw denormalized data for use in Druid.
 Very small lookups (count of keys on the order of a few dozen to a few hundred) can be passed at query time as a "map" 
 lookup as per [dimension specs](../querying/dimensionspecs.html).
 
+## Configuration
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.lookup.snapshotWorkingDir`| Working path used to store snapshot of current lookup configuration, leaving this property null will disable snapshot/bootstrap utility|null|
```

Comment:
thought about it but felt like ppls might chose to not use it. i can set it as mandatory if you feel that we can not use Lookup without this snapshot utility , which in MO not true.